src/analysis/locally_convex/with_seminorms.lean

```diff
@@ -524,3 +524,39 @@ begin
 end
 
 end topological_constructions
+
+section topological_properties
+
+variables [nontrivially_normed_field 𝕜] [add_comm_group E] [module 𝕜 E] [nonempty ι] [countable ι]
+variables {p : seminorm_family 𝕜 E ι}
+variables [uniform_space E]
+
+lemma with_seminorms.has_countable_basis (hp : with_seminorms p) :
+  (𝓝 (0 : E)).has_countable_basis (λ _, true)
+    (λ i : finset ι × ℕ, ball (i.fst.sup p) 0 (i.snd + 1)⁻¹) :=
```

Comment:
Do you care about the specific basis or do you only use that for showing that this filter is countably generated. If the latter, you could simply use `with_seminorms_iff_nhds_eq_infi` and then apply `filter.comap.is_countably_generated` and `filter.infi.is_countably_generated`

Reply:
I am not entirely certain that the first lemma is not used anywhere, but it does not feel to useful.